kratos/python_scripts/read_materials_process.py

```diff
@@ -132,7 +133,7 @@ def _AssignPropertyBlock(self, data):
 
         # Set the CONSTITUTIVE_LAW for the current properties.
         if "Variables" in mat["constitutive_law"].keys(): #pass the list of variables when constructing the constitutive law
-           constitutive_law = self._GetVariable( mat["constitutive_law"]["name"].GetString())(mat["constitutive_law"]["Variables"])
+           constitutive_law = self._GetConstitutiveLaw( mat["constitutive_law"]["name"].GetString())(mat["constitutive_law"]["Variables"])
```

Comment:
@RiccardoRossi I just tested this, it does not work.
I don't see how a constitutive law cold be constructed with passing Variables. There is no constructor for this

I would remove it if you are ok?